pkg/apis/api/types.go

```diff
@@ -762,6 +762,130 @@ type RiskLevel struct {
 	Level int
 }
 
+type ComponentReportRequestReleaseOptions struct {
+	Release string
+	Start   time.Time
+	End     time.Time
+}
+
+type ComponentReportRequestTestIdentificationOptions struct {
+	Component  string
+	Capability string
+	TestID     string
+}
+
+type ComponentReportRequestExcludeOptions struct {
+	ExcludePlatforms string
```

Comment:
`string` is ambiguous. What exactly are all of these? (i.e., what form?) and series of strings separated by commas, brackets, spaces?  Please add a comment.